src/restore.c

```diff
@@ -153,13 +153,58 @@ restore_main(int32_t argc, char **argv)
 	}
 
 	uint32_t line_no;
+	as_vector directories;
+	as_vector_init(&directories, sizeof(char*), 1);
+
+	// restoring from multiple directories
+	if (conf.directory_list != NULL) {
+
+		uint32_t dir_count = 0;
+
+		char *dir_clone = safe_strdup(conf.directory_list);
+		split_string(dir_clone, ',', false, &directories);
+		dir_count = directories.size;
+
+		for (uint32_t i = 0; i < dir_count; i++) {
```

Comment:
```suggestion
		char *dir_clone = safe_strdup(conf.directory_list);
		split_string(dir_clone, ',', false, &directories);
		uint32_t dir_count = directories.size;

		for (uint32_t i = 0; i < dir_count; i++) {
```
Do we need dir_count anyway?

Reply:
No, removed it.